README.md

```diff
@@ -65,6 +65,10 @@ The following sakai.properties are relevant to the Turnitin LTI integration:
 * This is the ID of the LTI tool you created above (likely "!turnitin)
 * There is no default for this property
 
+##### turnitin.ltiCreate
```

Comment:
I would make it clear this is the setting for the global tool and call it turnitin.globalLtiCreate or similar